Marlin/src/module/stepper_indirection.h

```diff
@@ -618,14 +618,13 @@ void reset_stepper_drivers();    // Called by settings.load / settings.reset
     #define    _REV_E_DIR(E)   do{ if (E == 0) { E0_DIR_WRITE( INVERT_E0_DIR); } else { E1_DIR_WRITE( INVERT_E1_DIR); } }while(0)
   #endif
 
-  #if EITHER(DUAL_X_CARRIAGE, MULTI_NOZZLE_DUPLICATION)
-    #define _DUPE(N,T,V)  do{ E##N##_##T##_WRITE(V); }while(0)
+  #if HAS_DUPLICATION_MODE
     #define NDIR(N) _DUPE(N, DIR,!INVERT_E##N##_DIR)
     #define RDIR(N) _DUPE(N, DIR, INVERT_E##N##_DIR)
     #define E_STEP_WRITE(E,V) do{ if (extruder_duplication_enabled) { DUPE(STEP,V); } else _E_STEP_WRITE(E,V); }while(0)
 
     #if E_STEPPERS > 2
-      #define _DUPE(N,T,V)    do{ if (duplication_e_mask <= (N)) E##N##_##T##_WRITE(V); }while(0)
+      #define _DUPE(N,T,V)    do{ if (TEST(duplication_e_mask, N)) E##N##_##T##_WRITE(V); }while(0)
```

Comment:
@InsanityAutomation — I'm pretty sure this is also now the case.

Reply:
I was looking at that line and debating what to do with it. I dont think the comparison will work. I dont have hardware to test 3 or 4 stepper duplication though at the moment... Ill have to look at the definition of the TEST macro tomorrow.

Reply:
It tests bit N.